### ISSUE TITLE
The laser isn't firing on Gerald

### ISSUE THREAD

**Issue by amcewen** [DoESLiverpool/somebody-should#109]
The laser isn't firing on Gerald - either when running a cut or when pressing the laser button.  It has been in use most of the day, so was working earlier - although it did have the same problem this morning and @DefProc played around with a few things (turning it off and on, downloading config, checking cut-out switch, etc.) and got it working again.

Have checked the coolant - temp is 24 degrees at the minute, and there's a reasonable flow through it (not quite as good as through Sophia's water pump, but it's a longer run of piping).
Also checked the safety interlock, and that seems okay.
Tried turning it off and on a few times, downloading config didn't seem to do anything (i.e. no feedback, but also didn't fix it).
Gave the tube a quick visual inspection and it seems okay.

**Comment by DefProc** [comment]
(Careful) multimeter checking on the laser tube power supply (:warning: high voltage in this area) revealed the WP input is high (4.93V) where Sophia is low (0V) at all times. A datasheet for a similar supply (http://www.jinlantrade.com/ebay/40W%20CO2%20laser%20power%20supply.pdf) suggest the WP input is likely connected to the water flow sensor.

**Comment by DefProc** [comment]
Lovely, the water sensor connector had a slightly loose female pin on the machine side. The pin (black wire) is now clipped into the connector properly and the laser is tested and working. :relieved:

**Comment by cheapjack** [comment]
Gerald stopped firing again today so amcewen and I opened him up and did some testing. It seems like the pin DefProc mentioned was still loose so we wired the flow sensor direct to the existing screw terminal. This didnt really help and we realised that the physical arrangement of the sensor may also have been restricting flow through the sensor itself. 

Further testing showed that the sensor was just not responding to the flow consistently at all and could possibly be faulty. It was not restricting flow of coolant just the switch plate inside but that needs further investigation in a new issue . For the time being we have bypassed the sensor; it is still plumbed in but we have removed the switch wiring and have just completed the circuit at the screw terminal where it meets the 2 white wires that eventually connect to G and WP on the power supply block.
